pkg/build/debian.go

```diff
@@ -27,8 +27,10 @@ func Debian(manifest model.Manifest) error {
 	if err := util.RunMake(manifest, "istio", nil, "sidecar.deb"); err != nil {
 		return fmt.Errorf("failed to build sidecar.deb: %v", err)
 	}
-	if err := util.CopyFile(path.Join(manifest.GoOutDir(), "istio-sidecar.deb"), path.Join(manifest.OutDir(), "deb", "istio-sidecar.deb")); err != nil {
-		return fmt.Errorf("failed to package istio-sidecar.deb: %v", err)
+	if util.FileExists(path.Join(manifest.RepoOutDir("istio"), "deb")) {
```

Comment:
this conditional is also not correct. If you want me to change the PR to read from both locations so the checks all pass, I can do that. If you want me to read from one place (as htis pr does with the absence of htis conditional) - the dryrun check will fail.

Reply:
let's just keep it simple and I'll merge this?

Reply:
i already pushed a change..  but put a hold on it ;)